templates/partials/individual-response-guidance.html

```diff
@@ -0,0 +1,51 @@
+{% from "components/collapsible/_macro.njk" import onsCollapsible %}
+
+{% if content.individual_response.hub_context %}
+  {% set title = _("If you can’t answer someone else’s questions") %}
+  {% set query_param = "?hub_context=true" %}
+{% else %}
+  {% set query_param = "?individual_id=" ~ content.individual_response.individual_id %}
+  {% set title = _("If you can’t answer questions for this person") %}
+{% endif %}
+
+{% set link_description =
+   _("If this is not possible, there are <a href=/individual-response%(query_param)s>other ways each person can do their own census.</a>", query_param=query_param)
```

Comment:
Yeah so originally I had tried this, but it didn't work. Looks like the issue was me using a set in line assignment instead of the set block operator. Works now 👍